tests/integration/backup_table_tests/test_backup_table_option.py

```diff
@@ -27,10 +27,16 @@ def check_backup_param_template(self, test_table_name, backup_is_expected):
         with open('target/run/test/models/{}.sql'.format(test_table_name), 'r') as ddl_file:
             ddl_statement = ddl_file.readlines()
             self.assertEqual('backup no' not in ' '.join(ddl_statement).lower(), backup_is_expected)
```

Comment:
I think I want this to be an exception that gets raised rather than an assertion since that'd be taken as a test....I think? Seems that way from the [docs](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertEqual) for UnitTest. A plain assert with a message explain use would be sufficient and not trigger a test failure but rather an outright error. Maybe that's overkill?

Do you have a opinion on which would be "best"?

Reply:
@VersusFacit While im not extremely familiar with the original purpose of using a configurable flag for assertion, I think that it was to handle circumstances where the state of the project overall (via defaults in yml, etc) conflict or differ with the configuration of an individual model. It is definitely a unique approach If we go the exception route we would have to most likely change some of the test models accordingly.

Reply:
Nope you're right. Sorry, we're making some changes to this testing framework and I'm still getting up to speed with (changing) best practices. Forgive the red herring.